include/lemlib/timer.hpp

```diff
@@ -41,7 +42,7 @@ class Timer {
          *
          * @param time time in milliseconds
```

Comment:
```suggestion
         * @param time time
```
get rid of unit in description